main/server/validations/index.js

```diff
@@ -0,0 +1,28 @@
+const { failure } = require('io-ts/lib/PathReporter')
+
+/**
+ * Validate and normalize the input (body) of the request.
+ * It ensures the body has the right model and does not include unnecessary attributes (they will be stripped)
+ *
+ * @template {A}
+ * @param {Decoder<A>} type
+ * @return {Function}
+ */
+function validateRequestInput(type) {
```

Comment:
This helper function helps us to validate input model so we are sure it respects what we expect in the request processing.